From fcccdf0185c213a21d712ee8700222de9feaa38e Mon Sep 17 00:00:00 2001
From: Steve deRosier <Steve.Derosier@lairdtech.com>
Date: Thu, 19 Sep 2013 16:46:35 -0700
Subject: [PATCH] defconfig: Add keyboard gpio driver

Adding CONFIG_KEYBOARD_GPIO to the wb45n defconfig. This driver is
necessary to wake the board from sleep using the IRQ button.  It can also
be used to trigger events based on the button being pressed.
---
 arch/arm/configs/wb45n_defconfig |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm/configs/wb45n_defconfig b/arch/arm/configs/wb45n_defconfig
index da121e0..76fec38 100644
--- a/arch/arm/configs/wb45n_defconfig
+++ b/arch/arm/configs/wb45n_defconfig
@@ -181,7 +181,8 @@ CONFIG_ATH6KL_DEBUG=y
 CONFIG_ATH6KL_REGDOMAIN=y
 # CONFIG_INPUT_MOUSEDEV is not set
 CONFIG_INPUT_EVDEV=y
-# CONFIG_INPUT_KEYBOARD is not set
+# CONFIG_KEYBOARD_ATKBD is not set
+CONFIG_KEYBOARD_GPIO=y
 # CONFIG_INPUT_MOUSE is not set
 # CONFIG_SERIO is not set
 CONFIG_LEGACY_PTY_COUNT=4
-- 
1.7.9.5

